test/python/test_select.py

```diff
@@ -104,3 +104,58 @@ def run_test(sess):
 
         assert (
             self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+
+class TestWhere(NgraphTest):
+    env_map = None
+
+    def __init__(self):
+        self.store_env_variables(['TF_OV_CONSTANT_FOLDING'])
```

Comment:
env_map = self.store_env_variables(['TF_OV_CONSTANT_FOLDING'])

Reply:
Fixed in #742